src/seedu/addressbook/commands/ChangeCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.Scanner;
+
+/**
+ * Changes a person phone number by using their index.
+ */
+public class ChangeCommand extends Command {
+
+    public static final String COMMAND_WORD = "change";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":Changes the phone number of a person in the addressbook. "
+            + "Example: " + COMMAND_WORD
+            + " John Doe 91234567";
+    public static final String MESSAGE_CHANGE_PERSON_SUCCESS = "Person number changed: %1$s";
+    public static final String MESSAGE_SUCCESS = "Person number changed: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+
+    private String newNumber;
+
+    public ChangeCommand(int targetVisibleIndex, String newNumber ) {
+        super(targetVisibleIndex);
+        this.newNumber = newNumber;
+    }
+
+
+
+
+    @Override
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.